added getAccuracyRegression function to calculate new accuracy score

BennettElisa opened the pull request:
1. renamed getAccuracy function to --> getAccuracyClassification
2. created getAccuracyRegression export function 
3. getAccuracyRegression function returns the evaluated result of getAverageDiff  / (max - min) values from the evaluated result of getRange 
4. updated conditionals within getSummaryStat function 
- if MLTypes.REGRESSION --> summaryStat.stat ---> getAccuracyRegression(state)
- if MLTypes.CLASSIFICATION --> summaryStat.stat ---> getAccuracyClassification(state)

Erin007 commented on src/redux.js:
```diff
@@ -532,19 +544,27 @@ export function getAccuracy(state) {
 
+export function getAccuracyRegression(state) {
+  console.log('inside getAccuracyR Function')
+  let returnValue = getRange(state, state.labelColumn)
```
I think we can call this variable `range` since `returnValue` is a little vague and might indicate it's the value `getAccuracyRegression` will return, which it's not.

BennettElisa replied:
ok great! I'll  change the variable name to `range`

Erin007 commented:
Awesome! Once you remove your notes/console.logs and rename that variable, it'll be looking really great! 

There's a merge conflict in `redux.js`. Have you seen this before? Basically means if you were to merge changes in your branch would interfere with changes already on the `main` branch. It usually happens when you're working off of a "stale" branch, meaning that your local branch is not up-to-date with the new changes that have merged to main since you last pulled. **In this case, looks like removing your notes will resolve the issue**, but let me know if it'd be helpful to walk through how to resolve merge conflicts more generally. 

Thanks for taking this little project on! We appreciate it.

BennettElisa commented:
Morning Erin! I changed the variable name and removed all the console.logs.
Hopefully, I got them all 😌. It would be great to better learn how to
resolve merge conflicts more generally and also for this particular pull
request as well. Is there a day/time that works for you in the upcoming
days for us to connect via zoom and talk through it?

Best,
Elisa

On Wed, Feb 10, 2021 at 12:22 PM Erin Bond <notifications@github.com> wrote:

> Awesome! Once you remove your notes/console.logs and rename that variable,
> it'll be looking really great!
>
> There's a merge conflict in redux.js. Have you seen this before?
> Basically means if you were to merge changes in your branch would interfere
> with changes already on the main branch. It usually happens when you're
> working off of a "stale" branch, meaning that your local branch is not
> up-to-date with the new changes that have merged to main since you last
> pulled. *In this case, looks like removing your notes will resolve the
> issue*, but let me know if it'd be helpful to walk through how to resolve
> merge conflicts more generally.
>
> Thanks for taking this little project on! We appreciate it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/code-dot-org/ml-playground/pull/69#issuecomment-776876431>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKMBKSQPIAHLAOUUFLXY5VLS6K6ERANCNFSM4XNEECLA>
> .
>


-- 

*Tell me, what is it you plan to do with your one wild and precious life? —
Mary Oliver*

Erin007 commented:
Thanks for the follow-up! There appear to be a bunch of un-related "cosmetic" changes to the redux.js file.  I suspect your IDE is set up to add trailing commas and add (). Could you remove these changes, please so that the diff is only the work related to the accuracy score work?

Erin007 reviewed (approved):
Awesome! You're ready to merge!

Erin007 commented:
fyi: @brondle - thanks for bringing this up!
